src/views/product-downloads-view/index.tsx

```diff
@@ -55,7 +56,7 @@ const ProductDownloadsViewContent = ({
   )
   const sidebarNavDataLevels = [
     generateTopLevelSidebarNavData(currentProduct.name),
-    generateProductLandingSidebarNavData(currentProduct),
+    generateInstallViewNavItems(currentProduct, sidebarMenuItems),
   ]
```

Comment:
I'm pretty sure we need 3 levels here now, since the install view's Sidebar is officially becoming unique from the product landing page's Sidebar. Otherwise, there is no way to navigate to a product's main pages in viewports <= `1000px` (achieved easily with page zoom, small screens, windows that are split or just not fullscreen, etc.).

```ts
const sidebarNavDataLevels = [
  generateTopLevelSidebarNavData(currentProduct.name),
  generateProductLandingSidebarNavData(currentProduct),
  generateInstallViewNavItems(currentProduct, sidebarMenuItems),
]
```

I think this also means that we don't want `generateInstallViewNavItems` to fallback to the landing page Sidebar items. What's your thought process behind having a fallback set of menu items? My current guess is so that there's not an error or weird behavior as we onboard more products?